jar/src/cljc/sixsq/slipstream/client/api/utils/http_utils.cljc

```diff
@@ -0,0 +1,16 @@
+(ns sixsq.slipstream.client.api.utils.http-utils)
+
+(def http-lib-insecure-key :kvlt.platform/insecure?)
+
+(defn- process-insecure
+  [req]
+  (if (contains? req :insecure?)
+    (-> req
```

Comment:
I think you can just get rid of the `if` statement and just use the threading expression as the body of the function.  It will always set the insecure keyword, but it will have the correct value of true or false/nil.